src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,28 @@
-export const TodoList = () => {};
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[],
+};
+
+export const TodoList: FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
```

Comment:
it's better to use semantic tags for list (`ul`, `li`)

Reply:
I agree) I've followed suit:
https://mate-academy.github.io/react_add-todo-form/